worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -555,76 +623,90 @@ public void scatterSchemBrush(Player player, InjectedValueAccess context,
         }
     }
 
-    @Command(name = "layer",
-             desc = "Replaces terrain with a layer.",
-             descFooter = "Replaces terrain with a layer.\n"
-                 + "Example: /br layer 5 95:1 95:2 35:15 - Places several layers on a surface\n"
-                 + "Pic: https://i.imgur.com/XV0vYoX.png")
+    @Command(
+        name = "layer",
+        desc = "Replaces terrain with a layer.",
+        descFooter = "Replaces terrain with a layer.\n"
+            + "Example: /br layer 5 95:1 95:2 35:15 - Places several layers on a surface\n"
```

Comment:
```suggestion
            + "Example: /br layer oak_planks orange_stained_glass magenta_stained_glass black_wool - Places several layers on a surface\n"
```
We should use modern material names as example